galgebra/printer.py

```diff
@@ -30,9 +29,9 @@
 Format_cnt = 0
 
 ip_cmds = \
-r"""
-$\\DeclareMathOperator{\Tr}{Tr}
-\\DeclareMathOperator{\Adj}{Adj}
+"""
+$\\DeclareMathOperator{\\Tr}{Tr}
```

Comment:
I was wrong to add the r prefix(break xpdf printing), I just need to fix the \T and \A without double backslashes.